src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1210,26 +1210,26 @@ impl<T: AstInfo> AstDisplay for CreateTypeStatement<T> {
         match &self.as_type {
             CreateTypeAs::List { options } => {
                 f.write_str(&self.as_type);
-                f.write_str("( ");
+                f.write_str("(");
                 if !options.is_empty() {
                     f.write_node(&display::comma_separated(options));
                 }
-                f.write_str(" )");
+                f.write_str(")");
             }
             CreateTypeAs::Map { options } => {
                 f.write_str(&self.as_type);
-                f.write_str("( ");
+                f.write_str("(");
                 if !options.is_empty() {
                     f.write_node(&display::comma_separated(options));
                 }
-                f.write_str(" )");
+                f.write_str(")");
             }
             CreateTypeAs::Record { column_defs } => {
-                f.write_str("( ");
+                f.write_str("(");
                 if !column_defs.is_empty() {
                     f.write_node(&display::comma_separated(column_defs));
                 }
-                f.write_str(" )");
+                f.write_str(")");
```

Comment:
Do we need a migration to update the existing types in the catalog?

Reply:
The existing definitions will still parse fine, they'll just look a little different so it's probably fine.

Reply:
Nevermind, it looks like by default on all upgrades we parse every catalog, convert them back to a string, and update their definition with the new string.

https://github.com/MaterializeInc/materialize/blob/eeac169932af04873f91e8e4b7429a7c26facc6e/src/adapter/src/catalog/migrate.rs#L23-L57